setup.py

```diff
@@ -68,6 +68,7 @@
     "diffusers": ["diffusers"],
     "quality": QUALITY_REQUIRE,
     "tests": TESTS_REQUIRE,
+    "tests-openvino": ["datasets[audio]>=1.4.0"],
```

Comment:
`datasets[audio]` is required for collecting calibration data for Whisper model. 

I had to add a separate test extra for this because when installing `datasets[audio]` in a general tests extra, INC tests fail. `datasets[audio]` require `numba` package which leads to INC tests failure.

Reply:
I would prefer if it's only installed in the openvino CI rather than have it being an extra

Reply:
also is all of `datasets[audio]` necessary ?

Reply:
> I would prefer if it's only installed in the openvino CI rather than have it being an extra

But this way if someone wants to run tests locally, it will be less clear why did they fail, because with this it is clearly stated that these requirements are needed for openvino testing and otherwise one would have to look into workflow files. Do you think it's worth it?

> also is all of datasets[audio] necessary ?

A regular `datasets` package is installed together with optimum-intel https://github.com/huggingface/optimum-intel/blob/main/setup.py#L33 . `datasets[audio]` additionally installs these these three packages https://github.com/huggingface/datasets/blob/main/setup.py#L144-L146 . The first two of them (`soundfile` and `librosa`) are necessary for the added test. The third one (`soxr`) is not. But I decided to install it as `datasets[audio]` as a safer option, for example in case some additional version requirements on `soundfile` or `librosa` will appear at `datasets[audio]`, or something like that.

Reply:
I prefer adding `datasets[audio]` to test require in general, and `tbb` as well, to fix numba issue https://github.com/intel/neural-compressor/issues/2031#issuecomment-2419904575

Reply:
I believe INC bit packing works differently whether `numba` package is installed or not https://github.com/intel/neural-compressor/blob/master/neural_compressor/torch/algorithms/weight_only/modules.py#L456-L457 . If we add a `numba` requirement to general tests requirement, then it will have some effect on INC tests, i.e. different methods will be called. This seems like a change that should be addressed in a separate PR.

Reply:
@IlyasMoutawwakil What do you think? 

I can open a separate PR after this one where we add `datasets[audio]` to general tests requirements and invite someone from INC side for a review.